app/src/main/java/ar/com/wolox/android/example/ui/home/fragment/HomeFragment.kt

```diff
@@ -0,0 +1,39 @@
+package ar.com.wolox.android.example.ui.home.fragment
+
+import android.graphics.Color
+import android.support.v4.app.Fragment
+import android.support.v4.util.Pair
+import ar.com.wolox.android.R
+import ar.com.wolox.android.example.ui.home.news.NewsFragment
+import ar.com.wolox.android.example.ui.home.profile.ProfileFragment
+import ar.com.wolox.wolmo.core.adapter.viewpager.SimpleFragmentPagerAdapter
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import ar.com.wolox.wolmo.core.presenter.BasePresenter
+import kotlinx.android.synthetic.main.fragment_home.*
+import kotlinx.android.synthetic.main.fragment_viewpager.vViewPager
+import javax.inject.Inject
+
+class HomeFragment @Inject constructor() : WolmoFragment<BasePresenter<Any>>() {
+
+    @Inject internal lateinit var page1Fragment: NewsFragment
```

Comment:
You can create a companion object in NewsFragment with a method newInstance(). So you only need to call NewsFragment.newInstance() when you need it.

Reply:
done!